js/components/image.js

```diff
@@ -200,11 +200,12 @@ Fliplet.FormBuilder.field('image', {
 
           var scaledImage = loadImage.scale(img, options);
           var imgBase64Url = scaledImage.toDataURL(mimeType, $vm.jpegQuality);
+          var flipletBase64Url = imgBase64Url + ';filename:' + file.name;
```

Comment:
Does this assume there will always be a file name and is that a valid assumption?

Or does this file set a default `file.name` somewhere?

Reply:
@tonytlwu https://developer.mozilla.org/en-US/docs/Web/API/File/name